src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int leftLeg;
+    private int rightLeg;
+
+    public RightTriangle(String color, int leftLeg, int rightLeg) {
+        super(color);
+        this.leftLeg = leftLeg;
+        this.rightLeg = rightLeg;
+    }
+
+    @Override
+    public double area() {

```

Comment:
You're extending a class to set behavior. This is not recommended. Instead, use an interface to set behavior. This is a code quality issue.